tests/unit_tests/test_lobbyconnection.py

```diff
@@ -313,6 +319,7 @@ async def test_command_game_join_calls_join_game(
     game.game_mode = "faf"
     game.id = 42
     game.name = "Test Game Name"
+    game.host = players.hosting
     game_service._games[42] = game
     lobbyconnection.player = players.hosting
```

Comment:
BTW this is set up a little weird because `players.hosting` is used both for the hosting player and the joining one